.github/pull_request_template.md

```
@@ -1,8 +1,7 @@
 ## Choose the appropriate template for your PR:
 
-### [Feature PR](./PULL_REQUEST_TEMPLATE/feature_template.md)
 
-#### Feature Request
+#### Feature/Bugfix Request
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)